powershell-adapter/Tests/powershellgroup.resource.tests.ps1

```diff
@@ -16,91 +16,121 @@ Describe 'PowerShell adapter resource tests' {
         $r = dsc resource list * -a *PowerShell*
         $LASTEXITCODE | Should -Be 0
         $resources = $r | ConvertFrom-Json
-        ($resources | ? {$_.Type -eq 'PSTestModule/TestClassResource'}).Count | Should -Be 1
+        ($resources | ? {$_.Type -eq 'TestClassResource/TestClassResource'}).Count | Should -Be 1
         ($resources | ? {$_.Type -eq 'PSTestModule/TestPSRepository'}).Count | Should -Be 1
     }
 
+    It 'Windows PowerShell adapter supports File resource' -Skip:(!$IsWindows){
+
+        $r = dsc resource list --adapter Microsoft.DSC/WindowsPowerShell
+        $LASTEXITCODE | Should -Be 0
+        $resources = $r | ConvertFrom-Json
+        ($resources | ? {$_.Type -eq 'PSDesiredStateConfiguration/File'}).Count | Should -Be 1
+    }
+
+    It 'Get works on Binary "File" resource' -Skip:(!$IsWindows){
+
+        $testFile = 'c:\test.txt'
```

Comment:
```suggestion
        $testFile = '$testdrive\test.txt'
```

Reply:
File resource does not support relative paths